sdcm/nemesis.py

```diff
@@ -1525,19 +1525,21 @@ def disrupt_mgmt_backup(self):
         self._mgmt_backup(backup_specific_tables=False)
 
     def _mgmt_backup(self, backup_specific_tables):
-        # TODO: When cloud backup is supported - add this to the below 'if' statement:
-        #  and not self.cluster.params.get('use_cloud_manager')
-        if not self.cluster.params.get('use_mgmt'):
+        if not (self.cluster.params.get('use_mgmt') or
+                self.cluster.params.get('use_cloud_manager')):
             raise UnsupportedNemesis('Scylla-manager configuration is not defined!')
-        if not self.cluster.params.get('backup_bucket_location'):
-            raise UnsupportedNemesis('backup bucket location configuration is not defined!')
 
         mgr_cluster = self.cluster.get_cluster_manager()
         cluster_backend = self.cluster.params.get('cluster_backend')
-        backup_bucket_location = self.cluster.params.get('backup_bucket_location')
-        if cluster_backend == 'aws':
+        if self.cluster.params.get('use_mgmt'):
+            if not self.cluster.params.get('backup_bucket_location'):
+                raise UnsupportedNemesis('backup bucket location configuration is not defined!')
+            backup_bucket_location = self.cluster.params.get('backup_bucket_location')
+        else:
+            backup_bucket_location = self.cluster.get_backup_buckets_name()
```

Comment:
won't it be clearer if you'll implement `get_backup_buckets_name` also for SCT on the cluster class ?
and then you check it's output and decide if nemesis should be skipped ?

Reply:
both cases, if we didn't figured what is the bucket we need to use, we can't run this nemesis